modular_skyrat/master_files/code/modules/clothing/under/akula_jobs.dm

```diff
@@ -5,28 +5,41 @@
 /// The proper layer to render the tail overlays onto
 #define TAIL_OVERLAY_LAYER 5.9
 
-/// A proc to be used in `equipped()` for all akula clothing which has the 'special tech' to keep their wearers slippery
-/obj/item/clothing/proc/apply_wetsuit_status_effect(mob/living/carbon/human/user)
+/datum/component/wetsuit
+	dupe_mode = COMPONENT_DUPE_UNIQUE
+
+/datum/component/wetsuit/RegisterWithParent()
+	. = ..()
+	RegisterSignal(parent, COMSIG_ITEM_EQUIPPED, PROC_REF(apply_wetsuit_status_effect))
+	RegisterSignal(parent, COMSIG_ITEM_DROPPED, PROC_REF(remove_wetsuit_status_effect))
```

Comment:
Just to be sure, this won't cause runtimes when the item is actually dropped? And does `COMSIG_ITEM_DROPPED` actually get called after it's unequipped from a slot?

Reply:
I tested this a bunch and didn't notice any runtimes, should be good.